32_communication_server/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-4bw+/aepP/YC94hEpVNVgiZdgIC5+VKNBQNGCHeKRQN+PtmoHDEXuppvnDJzQIu9" crossorigin="anonymous">
+    <title>32 Communication width the server</title>
+    <script src="app.js" async></script>
+</head>
+<body>
+    <div class="container">
+        <h1 class="mb-3">Список справ</h1>
+        <div class="row mb-3">
+            <div class="col-auto">
+                <input id="todoText" type="text" class="form-control" placeholder="нова справа">

```

Comment:
don't use capital letter in name of html classes and ids

Reply:
Ok